sdk/translation/ai-translation-text-rest/samples-dev/breakSentenceWithAutoDetection.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary This sample demonstrates how to make a simple call to the Azure Text Translator service to get sentences' boundaries.
+ */
+import TextTranslationClient, {
+  ErrorResponseOutput,
+  TranslatorCredential,
+  InputTextItem,
+  BreakSentenceItemOutput,
+  isUnexpected,
+} from "@azure-rest/ai-translation-text";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint = process.env["ENDPOINT"] || "https://api.cognitive.microsofttranslator.com";
+const apiKey = process.env["TEXT_TRANSLATOR_API_KEY"] || "<api key>";
+const region = process.env["TEXT_TRANSLATOR_REGION"] || "<region>";
+
+export async function main() {
+  console.log("== Sentence Boundaries with auto-detection sample ==");
+
+  const translateCedential = new TranslatorCredential(apiKey, region);
+  const translationClient = TextTranslationClient(endpoint, translateCedential, undefined);
+
+  const inputText: InputTextItem[] = [{ text: "How are you? I am fine. What did you do today?" }];
+  const breakSentenceResponse = await translationClient.path("/breaksentence").post({
+    body: inputText,
+  });
+
+  if (breakSentenceResponse.status !== "200") {
+    const error = breakSentenceResponse.body as ErrorResponseOutput;
+    throw error.error;
+  }
+
+  if (isUnexpected(breakSentenceResponse)) {
+    throw breakSentenceResponse.body;
+  }
+
+  const breakSentences = breakSentenceResponse.body as BreakSentenceItemOutput[];
```

Comment:
same comment about the explicit cast. We should see no explicit cast in our samples.

Reply:
This comment applies throughout the samples

Reply:
done